### ISSUE TITLE
2.0.0-beta-1 does not connect to server

### ISSUE THREAD

**Issue by niclan** [nikclayton/android-squeezer#613]
Hi,

I can't get Squeezer 2.0.0-beta-1 nor the older version to connect to my squeeze server on the Local LAN. It works just fine from any laptop and on a older phone it also works.

The LMS is not password protected. The version is `Logitech Media Server Versjon: 7.9.1 - 1522157629 @ Fri Mar 30 12:28:01 CEST 2018`

Broken phone: Samsung S7 running Android 9.
Working phone: Samsung A3 2017 running Android 7.

I just read the full thread on #611 - no other ports helped.

I also cannot get firefox or chrome on the phone to connect to the Web GUI.  However "MediaHouse-Pro" can connect to our local DLNA server and controll our TV - all on the same LAN.

The LAN and WIFI is bridged - one IP range, one broadcast domain.

Any clues?

**Comment by kaaholst** [comment]
Hi Nicolai,

Since it works on an old phone, it indicates that the problem is on the new phone. However these problems usually are on LMS or the computer it runs.

You can try to connect to LMS Web GUI from the computer on which LMS runs, and from another computer (on the same LAN as the phone you try to connect using Squeezer). If that works, then the URL is the exact same you should try in Squeezer and from a browser on the phone.

Also here might be some clues:
https://github.com/nikclayton/android-squeezer/issues/91#issuecomment-569967598

Squeezer runs fine on android 9, so this is not the issue. I don't know about Samsung S7, but as far as I know, we have no reports about it, so I think it should be fine.

I do not know what "The LAN and WIFI is bridged" means, so I can't help with that.

**Comment by niclan** [comment]
On the new phone I can't get firefox or chrome to connect to the GUI either so it's not specific to the app. It sort of looks like it's blocked in the phone.

Since the old phone works the https://github.com/nikclayton/android-squeezer/issues/91#issuecomment-569967598 thing does not seem to apply.

**Comment by kaaholst** [comment]
Yes I noticed you mentioned that it doesn't connect from a browser on the phone That's why I suggested to try to connect from another device, to check if the problem is limited to only the new phone. You might also want to check the advanced/security/Block Incoming Connections settings on LMS, to see if it's disabled or the new phones IP-address is allowed.
How to find the IP-address on Android differs, but its something like Settings/Network and Internet/Connection/Wi-Fi-settings/Advanced.

As I understand the comment from issue 91, the problem was that the server mistakenly thinks that the connecting device is not on the same network as the server, which is not allowed when the server is not password protected. So setting a username/password allows the connection even if the server thinks it's from another network.

**Comment by niclan** [comment]
OK, I've had a interesting evening. I finally decided to pursue this since I got some new wifi network gear at home and wanted _everything_ to be working.

In the end I have - quite a few months - managed to get my new phones MAC (ethernet address) into the configuration of one of the virtual machines I have in the network - cut and paste error seems to be the only way to do this.  And then I had configured that MAC in the firewall DHCP to give this server the ip 192.168.1.19.  So now the DHCP server would give the 192.168.1.19 address to my phone.

The result is that I could reach _no_ services on the host that runs my virtual machines. I.e. squeezer and the network management software.  Whereas all the UPnP stuff runs on another server so that worked just fine.

After installing some ping tool on my phone I noticed that the IP of the phone looked all wrong and managed to pull that thread until I got this resolved.

Now the squeezer app works very well thanks a lot!
